test/integration/scheduler_perf/util.go

```diff
@@ -396,11 +427,75 @@ func (tc *throughputCollector) init() error {
 }
 
 func (tc *throughputCollector) run(tCtx ktesting.TContext) {
-	podsScheduled, _, err := getScheduledPods(tc.podInformer, tc.namespaces...)
+	// The collector is based on informer cache events instead of periodically listing pods because:
+	// - polling causes more overhead
```

Comment:
Is it tested on the fastest workloads? Like `SchedulingBasic`?

Reply:
I compared the current master (= e456fbfaa64) against the latest revision of this PR (= 0f47ac56907). The short `SchedulingBasic` workloads complete too fast to gather throughput. For the longer ones there is no significant difference:

```
                                                      │     before      │                 after                 │
                                                      │ runtime_seconds │ runtime_seconds  vs base              │
PerfScheduling/SchedulingBasic/5Nodes-36                    12.36 ± 11%       12.34 ± 11%       ~ (p=0.677 n=6)
PerfScheduling/SchedulingBasic/500Nodes-36                  13.92 ±  1%       13.93 ±  1%       ~ (p=0.565 n=6)
PerfScheduling/SchedulingBasic/5000Nodes-36                 28.06 ±  0%       28.11 ±  1%       ~ (p=0.394 n=6)
PerfScheduling/SchedulingBasic/5000Nodes_10000Pods-36       42.52 ±  3%       42.69 ±  3%       ~ (p=0.310 n=6)
geomean                                                     21.29             21.31        +0.09%

...

                                                      │            before            │                       after                        │
                                                      │ SchedulingThroughput/Average │ SchedulingThroughput/Average  vs base              │
PerfScheduling/SchedulingBasic/5000Nodes-36                               397.2 ± 4%                     389.7 ± 6%       ~ (p=0.509 n=6)
PerfScheduling/SchedulingBasic/5000Nodes_10000Pods-36                     609.7 ± 3%                     609.0 ± 4%       ~ (p=0.394 n=6)
geomean                                                                   492.1                          487.1       -1.02%

                                                      │           before            │                       after                       │
                                                      │ SchedulingThroughput/Perc50 │ SchedulingThroughput/Perc50  vs base              │
PerfScheduling/SchedulingBasic/5000Nodes-36                              397.2 ± 4%                    389.7 ± 6%       ~ (p=0.509 n=6)
PerfScheduling/SchedulingBasic/5000Nodes_10000Pods-36                    610.0 ± 2%                    605.0 ± 2%       ~ (p=0.589 n=6)
geomean                                                                  492.3                         485.6       -1.36%

                                                      │           before            │                       after                       │
                                                      │ SchedulingThroughput/Perc90 │ SchedulingThroughput/Perc90  vs base              │
PerfScheduling/SchedulingBasic/5000Nodes-36                              397.2 ± 4%                    389.7 ± 6%       ~ (p=0.509 n=6)
PerfScheduling/SchedulingBasic/5000Nodes_10000Pods-36                    643.4 ± 2%                    639.4 ± 3%       ~ (p=0.699 n=6)
geomean                                                                  505.6                         499.2       -1.26%

                                                      │           before            │                       after                       │
                                                      │ SchedulingThroughput/Perc95 │ SchedulingThroughput/Perc95  vs base              │
PerfScheduling/SchedulingBasic/5000Nodes-36                              397.2 ± 4%                    389.7 ± 6%       ~ (p=0.509 n=6)
PerfScheduling/SchedulingBasic/5000Nodes_10000Pods-36                    804.8 ± 2%                    794.6 ± 3%       ~ (p=0.699 n=6)
geomean                                                                  565.4                         556.5       -1.58%

                                                      │           before            │                       after                       │
                                                      │ SchedulingThroughput/Perc99 │ SchedulingThroughput/Perc99  vs base              │
PerfScheduling/SchedulingBasic/5000Nodes-36                              397.2 ± 4%                    389.7 ± 6%       ~ (p=0.509 n=6)
PerfScheduling/SchedulingBasic/5000Nodes_10000Pods-36                    804.8 ± 2%                    794.6 ± 3%       ~ (p=0.699 n=6)
geomean                                                                  565.4                         556.5       -1.58%
```

Reply:
Great. We need to revisit the throughput thresholds some time later, as this change can affect the results a bit.